app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/representee/summary.scala.html

```diff
@@ -48,5 +49,11 @@
         "personRepresented"
     )
 
+    answers.fold(
```

Comment:
you don't need to `fold` on `CompleteRepresenteeAnswers` here - it's always going to go to the right